tests/tests/hafs_regional_atm_thompson_gfdlsf

```diff
@@ -0,0 +1,98 @@
+###############################################################################
+#
+#  HAFS regional atmosphere only
+#
+###############################################################################
+
+export TEST_DESCR="Compare HAFS regional atmosphere only results with previous trunk version"
```

Comment:
@ChunxiZhang-NOAA, also please update this line into the following or something similar. Thanks!
export TEST_DESCR="Compare HAFS regional atmosphere only results using the Thompson MP and GFDL surface layer schemes with previous trunk version"

Reply:
Changed to "Compare the results from HAFS regional atmosphere only using the Thompson microphysics scheme and GFDL surface layer scheme with previous trunk version".